content/lang/fr/404.md

```diff
@@ -1,5 +1,5 @@
-# Error
+# Erreur
 
-Yikes, looks like that page vanished.
+Yikes, ressemble à cette page disparue.
 
-Time to head [Home](/).
+Le temps de diriger [Accueil] (/).
```

Comment:
I understand that you‘d want to use Google Translate but these two translations didn’t actually mean anything in French :>